examples/bigcommerce/README.md

```diff
@@ -125,6 +80,30 @@ BIGCOMMERCE_ACCESS_TOKEN=5lw9ulikcp186tjgg3rs39kh4fg3vci
 
       Look for the ellipsis menu in the left toolbar again and drop the Product price, Product name, and Add to cart button into Makeswift.
 
+4. Using your own BigCommerce store
```

Comment:
I think this could be it's own section since it is related to the vercel deployment option as well. 

Just make it a ## and have one section with a screen shot of vercel dashboard env and then the section you have starting at 93 for the local section